CHANGELOG.md

```diff
@@ -1,18 +1,35 @@
-# [Next]
-
-## Highlights
-
--   start command gets more flexible by allowing to start only individual scripts.
--   stop command added that allows stopping individual processes
--   start is not blocking anymore (a start-daemon command is now the long running proccess)
--   pm config file isn't configurable anymore
--   waitOn support directly in script definition
--   renamed binary to dev-pm
--   added --follow option to start/restart command to follow logs after start
--   add cpu/memory usage to status output
--   add --interval option to status output to keep status open and optionally refresh periodically
--   add exponential growing backoff wait time between restarts when a process crashes
-
-# 1.0.0
+# Changelog
+
+## 2.0.0
+
+_TBD_
+
+This release marks a complete rewrite of the dev-process-manager.
```

Comment:
While many change have been made, including refactorings that moved most of the code around, this was technically not a rewrite.

Reply:
You're right, I've removed it.